src/Graphics/Texture.cs

```diff
@@ -97,6 +97,13 @@ internal static int GetFormatSize(SurfaceFormat format)
 			}
 		}
 
+		internal static int GetPixelStoreAlignment(SurfaceFormat format) {
```

Comment:
Brace on left and /**/, then this is set to go! (You can sign this comment if you want, too)